buttons.css

```diff
@@ -0,0 +1,19 @@
+.btn-lightblue {
+    background-color: var(--lightblue);
+    border-color: var(--navy);
+    color: var(--navy);
+    margin-right: 0.3em;
+}
+
+.btn-blue {
+    border-color: var(--blue);
+    background-color: var(--blue);
+    color: var(--white);
+}
+
+.btn {
```

Comment:
Nitpick: the base variant should be first in the file.

Because when you start reading it, you want to see the base/common one first, then continue with the more specific variants.